src/bridge/mgmtscripts/add_reactions.py

```diff
@@ -0,0 +1,75 @@
+import sys
+import yaml
+import time
+
+from werkzeug.security import generate_password_hash
+import rethinkdb as r
+
+from rethinkdb.errors import RqlRuntimeError, RqlDriverError
+
+if len(sys.argv) != 2:
+    print("Hey, thats not how you launch this...")
+    print("%s <config file>") % sys.argv[0]
+    sys.exit(1)
+
+# Open Config File and Parse Config Data
+configfile = sys.argv[1]
+cfh = open(configfile, "r")
+config = yaml.safe_load(cfh)
+cfh.close()
+
+# Establish Connection
+database = config['rethink_db']
+
+try:
+    if config['rethink_authkey']:
+        conn = r.connect(
+            host=config['rethink_host'], port=config['rethink_port'],
+            auth_key=config['rethink_authkey'], db=config['rethink_db']).repl()
+    else:
+        conn = r.connect(
+            host=config['rethink_host'], port=config['rethink_port'],
+            db=config['rethink_db']).repl()
+    print "Connecting to RethinkDB"
+except RqlDriverError:
```

Comment:
RqlDriverError message. RethinkDB not running is the most common but not the only reason when the script can't connect to DB, right?